db/seeds/memberships-permissions.rb

```diff
@@ -0,0 +1,35 @@
+custody_suite = Organisation.where(slug: "custody-suite").first
+call_centre = Organisation.where(slug: "capita").first
+laa = Organisation.where(slug: "laa").first
+law_firm_1 = Organisation.where(slug: "law-firm-1").first
+
+(1..5).each do |i|
+  cso = Membership.create(
```

Comment:
Do we want to use `first_or_create?` so we can run seeds without either: explosions or duplicate Memberships?